libraries/lib-datahandler/src/main/thrift/licenses.thrift

```diff
@@ -271,4 +273,8 @@ service LicenseService {
      * return filled todo
      **/
     Todo getTodoById( 1: string id);
+
+    list<CustomProperties> getCustomProperties(1: string documentType);
+
+    RequestStatus updateCustomProperties(1: CustomProperties customProperties, 2: User user )
```

Comment:
super nitpicking, pls consider whitespace before brackets close (because the other brackets are this way)